hipcheck/src/policy_exprs/expr.rs

```diff
@@ -49,6 +50,12 @@ pub enum Primitive {
 
 	/// Boolean.
 	Bool(bool),
+
+	/// Date-time.
+	DateTime(Zoned),
+
+	/// Span of time
+	Span(Span),
```

Comment:
Could you update these doc comments to specify the library the types come from and a summary of the string format they expect to parse from (I heard something about no fractional seconds?)

Reply:
And also specify that it's always interpreted in UTC (if that is indeed the case)

Reply:
You can have fractional seconds, but not fractional minutes or hours (which ISO 8601 allows).

The time zone is always set to UTC by the parser to simplify things, but you can specify an offset that the parser will use to adjust the time to the correct UTC time. e.g.  {foo}T9:30-05 will parse to {foo}T14:30["UTC"].

Reply:
That sounds good. I just want to make sure people reading the API docs can get a clue as to what format is expected and why the string they pass is getting rejected

Reply:
Put some text in the comments. Let me know if it makes sense.